src/inspect_ai/model/_providers/util/converse_model.py

```diff
@@ -0,0 +1,197 @@
+from typing import Any, Literal, Union
```

Comment:
Feels like this should be in the same source file as the result of bedock? (since its not really a util used by the rest of the providers)